### DISCUSSION TITLE
Newer versions of logbook not working

### DISCUSSION THREAD

**Discussion by rohithaksha** [zalando/logbook#1714] [Q&A]
I tried using logbook v 3.7.1 on spring boot 2.7.15 and Java 8. It’s not logging anything. When I switch to v 1.9.0 it works fine. I tried few other 3.x.x versions without any luck. Any idea what could be the issue?

**Comment by DuartePessoaGodinho** [comment]
Same here with Spring Boot on 2.5 does not work with LogBook 3.7.2. Haven't still found the highest working version.
